pages/_app.tsx

```diff
@@ -10,6 +11,15 @@ import { NotificationsProvider } from '../src/context/NotificationsContext'
 import { QuizProvider } from '../src/context/QuizContext'
 
 function MyApp({ Component, pageProps }: AppProps): JSX.Element {
+  // Check cookies haven't already been selected and if so don't show banner
+  useEffect(() => {
+    const isHotjarCookiesAccepted = localStorage.getItem(
+      'hotjarCookiesAccepted'
+    )
+    if (isHotjarCookiesAccepted) {
```

Comment:
I think we have to check `if (isHotjarCookiesAccepted === 'false')` here. Local storage only stores strings, and any string with a length greater than 0 is considered `true`,

Reply:
Good spot, I think it needs to be `if (isHotjarCookiesAccepted === 'true')`, as I only want that running when they say yes to cookies.